library/alloc/src/rc.rs

```diff
@@ -35,15 +35,26 @@
 //! `Rc<T>` automatically dereferences to `T` (via the [`Deref`] trait),
 //! so you can call `T`'s methods on a value of type [`Rc<T>`][`Rc`]. To avoid name
 //! clashes with `T`'s methods, the methods of [`Rc<T>`][`Rc`] itself are associated
-//! functions, called using function-like syntax:
+//! functions, called using [fully qualified syntax]:
```

Comment:
Do we actually call this "fully qualified syntax"? I thought that would be reserved for `<RC>::downgrade` form. 

Naively, "function like syntax" or "associated function syntax" seem like better options to me here.

Reply:
Yes, "fully-qualified sytax" is used for the form with the `<>`s. I would drop the extra clause, and just end after "functions."

Reply:
Hmm, wouldn’t it be better to give the readers somewhere to read more about fully qualified syntax? Side note, shouldn’t it be “fully-qualified syntax” with a hyphen?

Reply:
> wouldn’t it be better to give the readers somewhere to read more about fully qualified syntax?

Maybe. I am purely thinking about the wording itself.

> Side note, shouldn’t it be “fully-qualified syntax” with a hyphen?

I don't think so; the hyphen would be used if it is confusing which part "qualified" is attached to, that is, to disambiguate between

    fully-qualified syntax
    fully qualified-syntax

but I don't think that the latter would be a common interpretation